### PR TITLE
[luci] Add interface for QuanizedModelVerifier TestGraphs

### PR DESCRIPTION
**Pull request by dayo09**
This commit introduces SimpleTestGraph interface class which represents a sample test graph
with simply-choosen shapes and values.

ONE-DCO-1.0-Signed-off-by: Dayoung Lee <dayg502@gmail.com>

-----

From #6546

### CODE REVIEW

File: compiler/luci/pass/src/QuantizedModelVerifier.test.cpp [hunk 1]
```diff
@@ -155,7 +155,19 @@ void set_minmax_to_non_const(loco::Graph *g, float min, float max)
   }
 }
 
-class LogisticTestGraph final : public luci::test::TestIOGraph
+/**
+ * @brief Simple Test Graph
+ * @note
+ * The simple test graph's nodes are initialized with
+ * simplely-choosen shapes and values.
```
Comment (jinevening) [inline compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:162]:
```suggestion
 * @note
 * The simple test graph's nodes are initialized with
 * simple shapes and values.
```

Or, I think this note can be removed at all.

Comment (dayo09) [reply compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:162]: @jinevening I wanted to include the meaning that those `SimpleTestGraph` is just one instance, not like TestIOGraph.
I was worried that the name doesn't imply enough, so I put that note.

Comment (dayo09) [reply compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:162]: So I may leave a note with the change :-D

**Review by jinevening — APPROVED** [review]
LGTM Thank you

**Comment by seanshpark** [comment]
@dayo09 ,
- there are two changes: introducing `SimpleTestGraph` and revisit existing classe parent
- this code is OK but as a practice, can you split to two PRs?
- the title and the body of this PR is the first one :)

**Comment by dayo09** [comment]
@seanshpark I understand your point.

> if you want, we can go as-is (as the change is not that big or difficult to read)

In that case, may I just change the title to express the change's object clearly?
I think bringing these changes together will be readable as it explains the purpose at once.

**Comment by seanshpark** [comment]
Please fix conflicts :)

**Review by jinevening — DISMISSED** [review]
LGTM

**Review by seanshpark — DISMISSED** [review]
LGTM

**Comment by seanshpark** [comment]
@dayo09 , please resolve conflict again..

**Comment by dayo09** [comment]
@seanshpark @jinevening Rebased :-D

**Review by seanshpark — DISMISSED** [review]
LGTM

**Comment by seanshpark** [comment]
I forgot to NOT MERGE another PR -_-;;;
Please rebase again.

**Review by seanshpark — DISMISSED** [review]
LGTM

**Comment by jinevening** [comment]
@dayo09 Can you resolve the build failure?

Let's merge this first.

**Comment by dayo09** [comment]
@seanshpark @jinevening Fixed. 😄

**Review by jinevening — APPROVED** [review]

**Review by seanshpark — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
at last ~

**Pull request merged**